test/controllers/trails_controller_test.rb

```diff
@@ -51,6 +51,30 @@ class Api::V1::TrailsControllerTest < ActionController::TestCase
                            last["activity_type_name"]
   end
 
+  test "trails have nested species" do
+    trail = Trail.create(name: "howdy hike",
+                         city: "Denver",
+                         state: "CO",
+                         country: "United States",
+                         lat: 10.323,
+                         lng: -103.23,
+                         description: "A darn good time",
+                         directions: "take a right")
+    Species.create(kingdom: "Animalia",
+                             trail_id: trail.id,
+                             scientific_name: "Parus caeruleus",
+                             photo_url: "http://media.eol.org/content/2014/12/28/22/94944_orig.jpg",
```

Comment:
Line is too long. [100/80]<br>Align the elements of a hash literal if they span more than one line.